.github/workflows/linux-eic-shell.yml

```diff
@@ -1233,3 +1235,16 @@ jobs:
           issue-number: ${{ github.event.pull_request.number }}
           body-file: capybara_${{ github.event.pull_request.number }}.md
           edit-mode: replace
+
+  cancel-container:
+    if: cancelled() || failure()
+    needs:
+    - trigger-container
```

Comment:
This likely `needs: deploy-docs` too to make sure it doesn't get skipped too early.